### ISSUE TITLE
🐞[BUG]: ofActionCompleted() (perhaps all ofAction* functions) silently ignore lists passed in as arrays

### ISSUE THREAD

**Issue by nvahalik** [ngxs/store#1561]
### Affected Package
ngxs/store

### Is this a regression?
No.

### Description

This code throws no errors, but does not work:

```
this.actions$.pipe(
  ofActionCompleted( [ ActionOne, ActionTwo ] )
).subscribe(); // Will not fire, but will not throw an error, either.
```

While this code:
```
this.actions$.pipe(
  ofActionCompleted(ActionOne, ActionTwo)
).subscribe(); // Will fire.
```

Works as expected.

## 🔬 Minimal Reproduction

See it in action here: https://github-xroruv.stackblitz.io

## 🔥 Exception or Error

This call should either error or accept the array.

## Environment

N/A

**Comment by chulphan** [comment]
ofActionDispatched has same issue as this issue.
I think only not fire anything, but working expected.
How do you think??

**Comment by chulphan** [comment]
I think it seems to behavior of spread opeator, when an arguments passed in ofAction* and then pass to ofActionOperator function.
For example, When we pass an array to ofAction* function and pass the array(received from ofAction*) to ofActionOperator then the passed array converted to array of array.

**Comment by markwhitfeld** [comment]
I think it would be best to throw an error here.
An API that can be used in too many ways can cause confusion, so best to guide the user.

**Comment by markwhitfeld** [comment]
Good news, this has been addressed in NGXS v3.7 which was released last week.
( see https://dev.to/ngxs/announcing-ngxs-3-7-1kbk for release notes )

I will close this issue. Please let us know if it is not resolved.
